release-notes/opensearch-release-notes-1.3.0.md

```diff
@@ -0,0 +1,637 @@
+
+
+### OpenSearch Ml Common
+* add anomaly localization implementation ([#103](https://github.com/opensearch-project/ml-commons/pull/103))
+* refactor ML task data model; add create ML task index method ([#116](https://github.com/opensearch-project/ml-commons/pull/116))
+* integration step 1 and 2 for anomaly localization ([#113](https://github.com/opensearch-project/ml-commons/pull/113))
+* anomaly localization integration step 3 ([#114](https://github.com/opensearch-project/ml-commons/pull/114))
+* support train ML model in either sync or async way ([#124](https://github.com/opensearch-project/ml-commons/pull/124))
+* anomaly localization integration step 4 and 5 ([#125](https://github.com/opensearch-project/ml-commons/pull/125))
+* add train and predict API ([#126](https://github.com/opensearch-project/ml-commons/pull/126))
+* add ML Model get API ([#117](https://github.com/opensearch-project/ml-commons/pull/117))
+* integrate tribuo anomaly detection based on libSVM ([#96](https://github.com/opensearch-project/ml-commons/pull/96))
+* add ML Delete model API ([#136](https://github.com/opensearch-project/ml-commons/pull/136))
+* add fixed in time rcf ([#138](https://github.com/opensearch-project/ml-commons/pull/138))
+* Add ML Model Search API ([#140](https://github.com/opensearch-project/ml-commons/pull/140))
+* add circuit breaker ([#142](https://github.com/opensearch-project/ml-commons/pull/142))
+* add batch RCF for non-time-series data ([#145](https://github.com/opensearch-project/ml-commons/pull/145))
+* Add ML Task GET/Delete API ([#146](https://github.com/opensearch-project/ml-commons/pull/146))
+* Add Search Task API and Refactor search actions and handlers ([#149](https://github.com/opensearch-project/ml-commons/pull/149))
+* add minimum top contributor candidate queue size ([#151](https://github.com/opensearch-project/ml-commons/pull/151))
+* add more stats: request/failure/model count on algo/action level ([#159](https://github.com/opensearch-project/ml-commons/pull/159))
+* add tasks API in Client ([#200](https://github.com/opensearch-project/ml-commons/pull/200))
+
+
+### OpenSearch Observability
+* Feature latest observability ([#509](https://github.com/opensearch-project/observability/pull/509))
+* Live tail - Event analytics  ([#494](https://github.com/opensearch-project/observability/pull/494))
+* Add Events Flyout and Correlate Traces with logs ([#493](https://github.com/opensearch-project/observability/pull/493))
+* Merge Application Analytics into main ([#454](https://github.com/opensearch-project/observability/pull/454))
+
+
+### OpenSearch Performance Analyzer
+* Add .whitesource configuration file ([#119](https://github.com/opensearch-project/performance-analyzer/pull/119))
+* Add support for OPENSEARCH_JAVA_HOME ([#133](https://github.com/opensearch-project/performance-analyzer/pull/133))
+* Adding auto backport ([#146](https://github.com/opensearch-project/performance-analyzer/pull/146))
```

Comment:
Where should this go? Anything with workflows should not be considered new features but rather infrastructure.

(most of this section here, actually)